tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -399,15 +399,15 @@ class TestGetDagRunsPaginationFilters(TestDagRunEndpoint):
     @parameterized.expand(
         [
             (
-                "api/v1/dags/TEST_DAG_ID/dagRuns?start_date_gte=2020-06-18T18:00:00+00:00",
+                "api/v1/dags/TEST_DAG_ID/dagRuns?start_date_gte=2020-06-18T18:00:00Z",
```

Comment:
```suggestion
                "api/v1/dags/TEST_DAG_ID/dagRuns?start_date_gte=2020-06-18T18:00:00Z",
```
I'm wondering if this is a breaking change?

Reply:
Though the documentation is clear on using `Z` instead of `00+00:00`

Reply:
Though the documentation is clear on using `Z` instead of `+00:00`